tests/unit/output/test_output_manager_reports.py

```
@@ -67,3 +69,75 @@ def test_output_manager_writes_reports(
+
+    stale_run = tmp_path / "run-20000101-000000-000000-9999"
+    stale_run.mkdir()
+    stale_marker = stale_run / ".active"
+    stale_marker.write_text("424242\n", encoding="utf-8")
+
+    for suffix in (1, 2, 3):
+        (tmp_path / f"run-20000101-00000{suffix}-000000-9999").mkdir()
+
+    monkeypatch.setattr(
+        "lintro.utils.output.manager.OutputManager._pid_is_active",
+        lambda self, pid: False,
```

Ruff (0.15.10)

[warning] Unused lambda argument: `self`

(ARG005)